src/Farfetch.LoadShedding/Limiters/IAdaptativeConcurrencyLimiter.cs

```diff
@@ -13,18 +13,20 @@ public interface IAdaptativeConcurrencyLimiter
         /// <summary>
         /// Responsible for managing the current request.
         /// </summary>
-        /// <param name="priority">The priority of the execution of the task.</param>
+        /// <param name="priority">The <see cref="Priority"/> of the execution of the task.</param>
         /// <param name="function">A function that represents the request that has been processed.</param>
+        /// <param name="method">The method of the execution of the task.</param>
         /// <param name="cancellationToken">A cancellation token is used to signal that the running operation should be stopped.</param>
         /// <returns></returns>
-        Task ExecuteAsync(Priority priority, Func<Task> function, CancellationToken cancellationToken = default);
+        Task ExecuteAsync(Priority priority, Func<Task> function, string method = null, CancellationToken cancellationToken = default);
```

Comment:
The method should not be passed here, it is the base lib and it can be used in other context (it is not exclusive for web requests)